main.nf

```diff
@@ -604,7 +604,9 @@ process dia_spectral_library_search {
         -in ${mzml_file} \\
         -tr ${lib_file.baseName}.pqp \\
         -sort_swath_maps \\
-        -tr_irt ${irt_file.baseName}.pqp \\
+        #-enable_ms1 ${params.enable_ms1} \\
```

Comment:
If you comment this line, the newline characters will not be found and the command stops here

Reply:
Uncommented.